master/internal/command/command_manager.go

```diff
@@ -33,9 +39,17 @@ func (c *commandManager) Receive(ctx *actor.Context) error {
 	case tasks.GenericCommandSpec:
 		taskID := model.NewTaskID()
 		jobID := model.NewJobID()
-		return createGenericCommandActor(
+		if err := createGenericCommandActor(
```

Comment:
`createGenericCommandActor` never returned an error, it just `ctx.Respond`'d it because an error returned would return from the actor and crash commands. Just moved the responding up a layer and let it return errors so we didn't accidentally return one one day